.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,127 @@
+name: integration_testing
+on:
+  repository_dispatch:
+    types: [ducktape-test-command]
+
+jobs:
```

Comment:
any way we can put this into a bash script and then run that from the workflow? That's great for debugging as well as to be able to run the same thing locally...

Reply:
we have that w/ internal tooling `vtools ducky ...` and `vtools punisher ...` 

i.e.: to perform a full chaos suite, should be `vtools deploy ... && vtools ducky ..` similar w/ randomize fault injection `vtools deploy .... && vtools punisher ... `